comp/swerve100/src/main/java/org/team100/frc2024/RobotContainer.java

```diff
@@ -267,6 +275,10 @@ public RobotContainer(TimedRobot robot) throws IOException {
         whileTrue(driverControl::never,
                 new DriveToWaypoint100(goal, m_drive, planner, drivePID, swerveKinodynamics));
 
+        //Drive With Profile
+        whileTrue(operatorControl::driveToNote,
+                new DriveWithProfile(() -> m_noteCamera.fieldRelativePose2d(), m_drive, dthetaController, swerveKinodynamics));
```

Comment:
instead of a trivial lambda you can just refer to the function:

m_noteCamera::fieldRelativePose2d